README.md

```diff
@@ -305,6 +297,42 @@ bazel build L1MetadataArray_test_cts
 bazel run L1MetadataArray_test_cts_gui
 ```
 
+### Using the local flow
+
+The local flow allows testing the build with locally built OpenROAD-flow-scripts.
+It is based on bazel `Make` targets, for more information on those, please refer to relevant [implementation](https://github.com/The-OpenROAD-Project/bazel-orfs?tab=readme-ov-file#make-targets) paragraph.
```

Comment:
is it necessary to use an absolute https path?

Reply:
Here's a fixup: https://github.com/The-OpenROAD-Project/bazel-orfs/pull/35